CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### v1.2.0
```

Comment:
This should not be a version in the feature branch (It is a version until it is in the release branch)